app/controllers/concerns/new_inprocess_controller.rb

```diff
@@ -9,7 +9,10 @@ def index
     @search = TransactionSearch::Searcher.new(TransactionSearch::ProductSearcher,
                                               TransactionSearch::DateRangeSearcher,
                                               TransactionSearch::OrderedForSearcher).search(order_details, @search_form)
-    @order_details = @search.order_details.preload(:order_status, :assigned_user)
+    @order_details = @search.order_details
+                            .includes(:order_status)
+                            .joins("LEFT OUTER JOIN users AS assigned_users ON assigned_users.id = order_details.assigned_user_id")
```

Comment:
@jhanggi This is a little inelegant, but it works.

Reply:
Yeah, maybe pull it into a scope on `OrderDetail` so that it's kind of hidden away from the controller?